### PR TITLE
use nuvla api for old jobs cleanup

### PR DESCRIPTION
**Pull request by konstan**

### CODE REVIEW

**Comment by konstan** [comment]
@0xbase12 
Is this one used? https://github.com/nuvla/job-engine/blob/master/code/src/scripts/job_restore.py
If no, then I suggest we remove it. If yes, I suggest to route it via API server as well. Let me know - I’ll make the required change. Thanks.

**Comment by 0xbase12** [comment]
Yes it's used as a restoration process if we loose the machine storing the
job queue. We can do a restore of ES from our snapshots, and then use this
script to regenerate the job queue.

**Comment by konstan** [comment]
Thanks. As this is a disaster recovery script it's OK for it to go directly to DB. I'll do nothing with it.  
Please review the PR.

The related changes to compose files are https://github.com/SixSq/nuvla-ops/pull/8 and https://github.com/nuvla/deployment/pull/31. The second one failed Travis build. I launched it again and will be checking the result.

**Comment by konstan** [comment]
@0xbase12 https://github.com/nuvla/deployment/pull/31 is good now.

File: code/src/nuvla/job/actions/jobs_cleanup.py [hunk 1]
```diff
@@ -8,24 +8,22 @@
...
-        query_old_completed_jobs = {'query': {'query_string': {'query': query_string}}}
-        result = self.es.delete_by_query(index='nuvla-job', body=query_old_completed_jobs)
-
-        if result['timed_out'] or result['failures']:
-            error_msg = 'Cleanup of completed jobs have some failures: {}.'.format(result)
-            logging.warning(error_msg)
-            self.job.set_status_message(error_msg)
+        days_back = 7
+        filter = f"(state='SUCCESS' or state='FAILED') and created<'now-{days_back}d'"
+        ret = self.nuvla.delete_bulk(index='job', filter=filter)
```
Comment (0xbase12) [inline code/src/nuvla/job/actions/jobs_cleanup.py:20]: Delete_bulk parameter resource-type is a positional argument, not a kwargs named index.

Comment (konstan) [reply code/src/nuvla/job/actions/jobs_cleanup.py]: Fixed in 1350a61

**Pull request merged**